guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.md

```diff
@@ -57,55 +57,63 @@ For all known {{site.data.var.ee}} 2.2.0 GA issues, see [Magento Commerce 2.2.0
 
 This release contains numerous fixes and enhancements. This section includes only fixes for {{site.data.var.ece}}. For all known {{site.data.var.ee}} 2.2.0 GA issues, see [Magento Commerce 2.2.0 Release Notes]({{ site.baseurl }}/guides/v2.2/release-notes/ReleaseNotes2.2.0EE.html).
 
-<!--RFC-125-->* We removed `var/view_preprocessed` symlinking to fix an issue that was causing JavaScript minification conflicts.
+<!--RFC-125-->
```

Comment:
All Cloud release notes have the ticket comment inline. Is it affecting the presentation? Or is it a convention thing that you are cleaning as you see it?

Reply:
This violates [rule MD006](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md006---consider-starting-bulleted-lists-at-the-beginning-of-the-line).

Reply:
Yes, it violates the rule and returns errors.